ui/shared/tabs/LogTab.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { LazyLog } from 'react-lazylog';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faExpand, faFileAlt } from '@fortawesome/free-solid-svg-icons';
+
+import TextLogStepModel from '../../models/textLogStep';
+import { getLogViewerUrl } from '../../helpers/url';
+
+const errorLinesCss = function errorLinesCss(logErrors) {
+  const style = document.createElement('style');
+  const rule = logErrors
+    .map(({ lineNumber }) => `a[id="${lineNumber}"]+span`)
+    .join(',')
+    .concat('{background:#fbe3e3;color:#a94442}');
+
+  style.type = 'text/css';
+  document.getElementsByTagName('head')[0].appendChild(style);
+  style.sheet.insertRule(rule);
+};
+
+class LogTab extends React.PureComponent {
```

Comment:
I wasn't sure whether to call this `LogTab` or `LogPanel`.  Thoughts?

Reply:
Additionally, I opted NOT to refactor the whole `LogViewer` to use this panel, since it's sort of a different animal.  But I could, after Suyash's PR lands.  It probably would be good to share some of that logic for colorizing, etc.  Or we could do that in a follow-up PR.

Reply:
Either LogviewerTab or logviewerPanel could work.